src/ViewModels/test/LoginViewModelTests.cs

```diff
@@ -440,17 +440,6 @@ public void NavigateToTargetPage_SetsPageNumTo1()
             Assert.AreEqual(1, _sut.PageNum);
         }
 
-        [TestMethod]
-        [TestCategory("CanProceedToAuthentication")]
-        [DataRow("https://some.legit.address")]
-        [DataRow("http://my.cool.api")]
-        public void CanProceedToAuthentication_ReturnsTrue(string targetApiAddress)
```

Comment:
What's the rationale behind getting rid of this test? Still seems like it'd be good to test unless I'm missing something?

Reply:
discussed and changed!